src/app/modules/OpenshiftStreams/components/StreamsTableConnected/useKafkaInstances.ts

```diff
@@ -196,3 +196,16 @@ function valuesToQuery(
     )
     .join(" or ");
 }
+
+function apiStatusToUIStatus(status: string): Status {
+  const mapping: { [key: string]: Status } = {
+    accepted: "accepted",
+    preparing: "preparing",
+    provisioning: "provisioning",
+    ready: "ready",
+    failed: "degraded",
+    deprovision: "deprovision",
+    deleting: "deleting",
+  };
```

Comment:
There are additional new statuses:
`suspending`, `suspended` and `resuming`. 

Please see the epic brief on kafka suspension or https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1346 and https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1344:

Reply:
When will that go live tho? In the UI we already have support for the `suspended` status, but not for the other two transient ones. I suppose we could show `suspending` as `suspended` as there is little value in showing that the instance is transitioning to that state, but the `resuming` one might require its own label. CC @jgiardino

Reply:
The two PRs have been merged from the control plane.
The feature is not yet complete as it requires this https://github.com/bf2fc6cc711aee1a0c2a/kas-fleetshard/pull/771 to be merged. 

I think it is worth showing the status as it is instead of mapping it as suspending means suspending and not suspended because the Kafka has not been suspended yet.. 

/cc @MikeEdgar

Reply:
Like @riccardo-forina, I was also thinking that `suspending` could map to the "Suspended" status in the UI, but you make a good point, @machi1990, that there is a difference in what functionality is available between `suspending` and `suspended`.

@riccardo-forina What are your thoughts on using the same styles we use for `deleting` for `suspending`. Could these styles more generally represent spinning something down?

And on the topic of `resuming` could our "Creating" styles be used more generally to represent something that we're spinning up?

@machi1990 Can you clarify what the sequence of statuses would be that include `resuming`? For example is it
`suspended` ➡️ `resuming` ➡️ `ready`
or are there other statuses that the instance would progress through before getting to `ready`?